CoffeeChat/Controller/OnboardingInterestsViewController.swift

```diff
@@ -60,21 +55,19 @@ class OnboardingInterestsViewController: UIViewController {
         titleLabel.font = ._24CircularStdMedium
         view.addSubview(titleLabel)
 
-        tableView.delegate = self
-        tableView.dataSource = self
-        tableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.reuseIdentifier)
-        tableView.isScrollEnabled = true
-        tableView.clipsToBounds = true
-        tableView.backgroundColor = .none
-        tableView.allowsMultipleSelection = true
-        tableView.bounces = false
-        tableView.showsHorizontalScrollIndicator = false
-        tableView.showsVerticalScrollIndicator = false
-        tableView.separatorStyle = .none
-        tableView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 30, right: 0)
-        view.addSubview(tableView)
-        view.addSubview(topFadeView)
-        view.addSubview(bottomFadeView)
+        fadeTableView.tableView.delegate = self
+        fadeTableView.tableView.dataSource = self
+        fadeTableView.tableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.reuseIdentifier)
+        fadeTableView.tableView.isScrollEnabled = true
+        fadeTableView.tableView.clipsToBounds = true
+        fadeTableView.tableView.backgroundColor = .none
+        fadeTableView.tableView.allowsMultipleSelection = true
+        fadeTableView.tableView.bounces = false
+        fadeTableView.tableView.showsHorizontalScrollIndicator = false
+        fadeTableView.tableView.showsVerticalScrollIndicator = false
+        fadeTableView.tableView.separatorStyle = .none
+        fadeTableView.tableView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 30, right: 0)
+        view.addSubview(fadeTableView)
```

Comment:
kinda piggybacking off of daniel's comment. looking at the two uses of the `fadeTableView` it seems like all of these configuration options are the same, so i think it makes sense to do default styling in the `FadeTableView`, that can be overridden if needed